core/src/main/java/com/scalar/db/storage/cassandra/DeleteStatementHandler.java

```diff
@@ -56,7 +57,9 @@ protected ResultSet execute(BoundStatement bound, Operation operation) {
 
   private com.datastax.driver.core.querybuilder.Delete prepare(Delete del) {
     com.datastax.driver.core.querybuilder.Delete delete =
-        QueryBuilder.delete().from(del.forNamespace().get(), del.forTable().get());
+        QueryBuilder.delete()
+            .from(
+                quoteIfNecessary(del.forNamespace().get()), quoteIfNecessary(del.forTable().get()));
     com.datastax.driver.core.querybuilder.Delete.Where where = delete.where();
 
     del.getPartitionKey().forEach(v -> where.and(QueryBuilder.eq(v.getName(), bindMarker())));
```

Comment:
Don't we need to quote `v.getName()`?

Reply:
Thanks for pointing it out. We have to quote all the columns in case it is a reserved keyword